CMakeLists.txt

```diff
@@ -171,6 +180,12 @@ install (
   FILES cmake/dateConfig.cmake "${version_config}"
   DESTINATION ${CONFIG_LOC})
 
+if ( BUILD_TZ_LIB )
```

Comment:
I'd think it's possible to still generate/install the `.pc` file even when `BUILD_TZ_LIB` is off, bit you'd just want to leave the `Libs:` line empty in that case. That said, if this works for your current use case, it's fine with me - we can always revisit later.

Reply:
Option documentation for `BUILD_TZ_LIB` states "build/install of TZ library". I believe that option is turned off in case if this library is directly included  in project as per recommendation. Though I'm not sure how this usually works, but for [Waybar](https://github.com/Alexays/Waybar/blob/591eb2ea38a7df53305fa39bb05492e1036c767e/subprojects/date.wrap) it looks like absence of `date.pc` is not a problem.